code/datums/brain_damage/magic.dm

```diff
@@ -25,7 +25,7 @@
 		return
 
 	if(COOLDOWN_FINISHED(src, damage_warning_cooldown))
-		to_chat(owner, span_warning("<b>The light burns you!</b>"))
+		to_chat(owner, span_userdanger("The light burns you!"))
```

Comment:
This was also undocumented upped to userdanger but I don't have a problem with it tbh

Reply:
I'm trying to keep some consistency with the context of spans. Warning is more of a "you tried this and it didn't work", danger is "you could be injured"

Reply:
If you're updating spans in general (ie, changing them to new spans) you should mention that (I thought this was just straight conversions pr)

Reply:
done - i've also tried to outline my thoughts on the changes for consistency.